creativecloud/scripts/miloUtils.js

```diff
@@ -0,0 +1,6 @@
+import { setLibs } from './utils.js';
```

Comment:
I would not create a specific file only for this.
this should go in https://github.com/npeltier/cc/blob/MWPW-136872/creativecloud/scripts/utils.js